src/main/java/svizhik/restapiproject/dto/Cat.java

```diff
@@ -0,0 +1,15 @@
+package svizhik.restapiproject.dto;
+import lombok.*;
+
+
+@Data
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Cat {
```

Comment:
После объявления класса должна быть пустая строка
```suggestion
public class Cat {

```